### DISCUSSION TITLE
Mobile Issues

### DISCUSSION THREAD

**Discussion by kunla** [andrew-codes/playnite-web#678] [Q&A]
I've been able to get the web app working on my local machine, but it doesn't work on my mobile devices (Android and iPhone). I've tried all the troubleshooting tips I could find, but nothing has worked. Does anyone have any suggestions?

I see around ~60% of my library on the desktop website but the mobile site is stuck on the "Library" page - clicking on the "My Games" icon does nothing.

**Comment by andrew-codes** [comment]
This looks related to access over LAN. See #705, releasing now. You can now disable csp.
